apps/web/src/components/search/index.js

```diff
@@ -16,12 +16,21 @@ GNU General Public License for more details.
 You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
-
 import * as Icon from "../icons";
 import "./search.css";
 import Field from "../field";
-
+import { useEffect } from "react";
+import { useStore } from "../../stores/theme-store";
 function SearchBox(props) {
+  const { theme } = useStore();
+  useEffect(() => {
+    const search = document.getElementById("search");
+    if (theme === "dark") {
+      search.classList.add("darkModeSearch");
+    } else {
+      search.classList.remove("darkModeSearch");
+    }
+  }, [theme]);
```

Comment:
There's no need to go to all this trouble. Just do inline styling via the sx prop. More details here: https://theme-ui.com/sx-prop